src/database/accessors/loadTableMetrics.ts

```diff
@@ -0,0 +1,27 @@
+import { db } from '../db';
+import { isTableMetrics } from '../../types';
+import { InternalValidationError } from '../../errors';
+import type { TableMetrics } from '../../types';
+
+export const loadTableMetrics = async (tableName: string): Promise<TableMetrics> => {
+  const metricsQueryResult = await db.query<TableMetrics>(`
+    SELECT COUNT(id) AS "count",
```

Comment:
5

See (2). It might be premature optimization to change this query to explicitly request an estimate, but maybe not. It's good as-is but maybe a comment in the code to note the potential future optimization if this query gets heavy.

Reply:
Can you say more here? I'm not sure I understand what you're referencing in [the comment you labeled as 2](https://github.com/PhilanthropyDataCommons/service/pull/296#discussion_r1164520000).

I think the complexity of asking for a [count estimate](https://wiki.postgresql.org/wiki/Count_estimate) is premature. If this becomes a bottleneck, we can discover that through the kind of performance analysis we'll need to be doing at that point anyway - a point I would expect to be far in the future.

Reply:
I would prefer to not leave a comment in the code but rather open an issue in our issue tracker.  Happy to open that issue, or also comfortable waiting until it becomes a bottleneck of some kind.